vfox-web/.angular-cli.json

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
I am NOT an AngularJS or Nodejs expert so I would need a code walk-thru some day to understand why we've 80+ files in Git when we just started this project! In fact, under vfox-web directory, I noticed that 37,579 files were 'generated'! Are these from Nodejs? Do we need all these?